rollup.config.js

```diff
@@ -0,0 +1,47 @@
+import commonjs from "rollup-plugin-commonjs";
+import nodeResolve from "rollup-plugin-node-resolve";
+
+export default {
+    input: "./lib/sinon.js",
+    output: {
+        file: "pkg/bundle-esm.js",
+        format: "esm"
```

Comment:
Just taking a quick look before getting a little time to test, but should the format be "es"? I don't see "esm" as an alias at https://rollupjs.org/guide/en

Reply:
Worked accidentally, I guess :smile: Still produced an EcmaScript module with `export default sinon`. Fixed in later commit. Thanks for noticing.

Reply:
What I suspect perhaps will cause trouble is the use of the `os` module ...

```
(!) Unresolved dependencies
https://github.com/rollup/rollup/wiki/Troubleshooting#treating-module-as-external-dependency
os (imported by node_modules/supports-color/index.js, commonjs-external:os)
```